include/common.h

```diff
@@ -44,6 +44,17 @@ unsigned int memtest(unsigned int start, unsigned int end);
 #define COL8_008484     14
 #define COL8_848484     15
 #define MEM_MAP_ADDR (char *)((0x00078004)-(SYSSEG << 4))
+#define MAX_FREE_MEM_BLOCK 10
+#define LOW_MEM 0x10000 //current igd base address                        
```

Comment:
What is IGD, I dump register info at your codebase. 
CS register is 0x10, it point to 0x10000.

```
(qemu) info  registers 
EAX=00000000 EBX=00000000 ECX=000ffebe EDX=0000000f
ESI=00000108 EDI=000080a7 EBP=000fffec ESP=000fff10
EIP=00001b71 EFL=00000206 [-----P-] CPL=0 II=0 A20=1 SMM=0 HLT=1
ES =0018 00010000 ffffffff 00cf9300 DPL=0 DS   [-WA]
CS =0010 00010000 ffffffff 00cf9a00 DPL=0 CS32 [-R-]
SS =0018 00010000 ffffffff 00cf9300 DPL=0 DS   [-WA]
DS =0018 00010000 ffffffff 00cf9300 DPL=0 DS   [-WA]
FS =0018 00010000 ffffffff 00cf9300 DPL=0 DS   [-WA]
GS =0018 00010000 ffffffff 00cf9300 DPL=0 DS   [-WA]
LDT=0000 00000000 0000ffff 00008200 DPL=0 LDT
TR =0000 00000000 0000ffff 00008b00 DPL=0 TSS32-busy
GDT=     000702fa 00000020
IDT=     0026f800 000007ff
CR0=00000011 CR2=00000000 CR3=00021000 CR4=00000000
DR0=0000000000000000 DR1=0000000000000000 DR2=0000000000000000 DR3=0000000000000000 
DR6=00000000ffff0ff0 DR7=0000000000000400
EFER=0000000000000000
FCW=037f FSW=0000 [ST=0] FTW=00 MXCSR=00001f80
FPR0=0000000000000000 0000 FPR1=0000000000000000 0000
FPR2=0000000000000000 0000 FPR3=0000000000000000 0000
FPR4=0000000000000000 0000 FPR5=0000000000000000 0000
FPR6=0000000000000000 0000 FPR7=0000000000000000 0000
XMM00=00000000000000000000000000000000 XMM01=00000000000000000000000000000000
XMM02=00000000000000000000000000000000 XMM03=00000000000000000000000000000000
XMM04=00000000000000000000000000000000 XMM05=00000000000000000000000000000000
XMM06=00000000000000000000000000000000 XMM07=00000000000000000000000000000000
```

Reply:
gdt base address, this not used now, i can remove it.